internal/controllers/pod_controller.go

```diff
@@ -144,17 +148,20 @@ func (r *PodReconciler) isValidationEnabledForNS(namespace string) bool {
 	if err := r.Get(context.TODO(), client.ObjectKey{Name: namespace}, &ns); err != nil {
 		return false
 	}
-	if ns.GetLabels()[pkg.NamespaceValidationLabel] != "enabled" {
-		return false
-	}
-	return true
+	return validate.IsValidationEnabledForNS(&ns)
 }
 
-func (r *PodReconciler) admitPodImage(image string) (string, error) {
-	err := r.Validator.Validate(image)
-	if err != nil {
-		return pkg.ValidationStatusFailed, err
+func labelForValidationResult(result validate.ValidationResult) string {
+	switch result {
+	case validate.NoAction:
+		return ""
```

Comment:
`NoAction` case clean label. Is it needed behavior?

Reply:
I used as indicator to not label. I changed the logic a little bit.